AoE2ScenarioParser/datasets/effects.py

```diff
@@ -693,6 +693,101 @@ class EffectId(IntEnum):
     **Version notice**: \n
     This effect was added in: 1.51 & Trigger Version 3.5
     """
+    SET_OBJECT_COST = 80
+    """Attributes for the **set_object_cost** effect are: \n
+    - object_list_unit_id
+    - source_player
+    - quantity
+    - tribute_list
+
+    **Version notice**: \n
+    This effect was added in: 1.54 & Trigger Version 3.5
```

Comment:
not sure what trigger version is. kept it as previous

Reply:
Ah, thanks, I'll update them to the newest one as they couldn't be used before